src/librustc_resolve/diagnostics.rs

```diff
@@ -1674,6 +1674,29 @@ fn main() {
 ```
 "##,
 
+E0910: r##"
```

Comment:
Is 909 the biggest error code so far? I could have sworn we're still <700.

Reply:
Oh, I'm off by two... The largest I found was 907: https://github.com/rust-lang/rust/blob/b1f8e6fb06d7362eeb2065347a7db94e76b1cb2f/src/librustc_typeck/diagnostics.rs#L4829

I wasn't really sure how to choose the error number, though...

Reply:
Tidy is supposed to tell you:
```
[00:05:36] * 534 error codes
[00:05:36] * highest error code: E0908
[00:05:36] * 175 features
```

I think that somebody sneaked a high error code skipping a bunch somewhere 👀

Reply:
Hmm... maybe if I run the full test suite? Let me try.

Reply:
Yep, it appears that tidy does not run for a full build...

Reply:
Ok, I changed it to E0909